web/js/random-passphrase.js

```diff
@@ -0,0 +1,5 @@
+import * as RandomPhrase from './diceware/random-phrase.js';
```

Comment:
Are we getting anything from having this extra function layer of `generate(..)`, as opposed to calling `RandomPhrase.get(..)` directly?